packages/astro/src/i18n/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { AstroConfig } from '../@types/astro.js';
+import type { Logger } from '../core/logger/core.js';
+
+/**
+ * The base URL
+ */
+export function getI18nBaseUrl(locale: string, config: AstroConfig, logger: Logger) {
```

Comment:
Is `getI18nBaseUrl` user facing? It would be hard for them to obtain a value for `config` and `logger`. If it's only for integrations, then maybe we should leave it as `astro/i18n` like `astro/errors`.

Reply:
Yes, that would the idea yes